tests/test.yml

```diff
@@ -0,0 +1,5 @@
+---
+- hosts: localhost
+  remote_user: root
+  roles:
+    - ansible-role-jenkins-cli/
```

Comment:
http://docs.ansible.com/ansible/playbooks_roles.html#embedding-modules-and-plugins-in-roles ! :ok_hand: